index.js

```diff
@@ -51,7 +60,9 @@ const mergeDefaults = (opts) => {
   const defaults = {
     keyword: null,
     redirectsLimit: 3,
-    userAgent: 'Generic curl robot',
+    headers: {
+      'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36',
```

Comment:
I'd like to avoid having a default that pretends to be something else. This is a library and if the user doesn't change it it should identify itself as one.


```suggestion
      'User-Agent': 'Uptime-check - https://www.npmjs.com/package/uptime-check',
```